.env-prod-enrollment-example

```diff
@@ -38,3 +38,6 @@ SHOW_DEMO_WARNING=true
 NEW_RELIC_LICENSE_KEY=<New Relic License Key>
 SENTRY_URL=<Sentry URL>
 JOB_RUN_EMAIL=<Email for job results>
+
+EXPORT_OUTER_BATCH_SIZE = 1000
+EXPORT_INNER_BATCH_SIZE = 200
```

Comment:
Should this be 500?

Reply:
This is just any example number, so doesn't need to be 500.